### PR TITLE
fix: dup category issue fixed

### PR DESCRIPTION
**Pull request by squrki**

### CODE REVIEW

**Review by dave-b-b — CHANGES_REQUESTED** [review]
@squrki  I don't understand what any of these changes are meant to fix.... Didn't we already fis the duplicate category issue? If we did fix the issues already, then can you close this?

**Comment by squrki** [comment]
There is still a conflict with the backend cat names. This fixes that.

File: client/src/components/CategoryModal.jsx [hunk 1]
```diff
@@ -46,8 +46,8 @@ const CategoryModal = () => {
         user.categories = selectedItems.map((item) => new Category(item));
         const tempUser = new User(user);
         setUser(tempUser);
+        localStorage.setItem('user', tempUser)
```
Comment (dave-b-b) [inline client/src/components/CategoryModal.jsx:49]: This is a good change.

File: client/src/components/ExpensesTable.jsx [hunk 2]
```diff
@@ -92,6 +98,7 @@ const ExpensesForm = () => {
             setOcrData(new Transaction());
         }
         setIsUpdating(false);
+        localStorage.setItem('user', result)
```
Comment (dave-b-b) [inline client/src/components/ExpensesTable.jsx:101]: I don't think this should get set until the response comes back from teh server.

File: client/src/models/User.js [hunk 3]
```diff
@@ -204,7 +204,8 @@ class User {
 function toProperCase(name) {
     if (!name) return;
     const lower = name.toLowerCase();
-    return lower.charAt(0).toUpperCase() + lower.slice(1);
+    // return lower.charAt(0).toUpperCase() + lower.slice(1);
+    return lower
```
Comment (dave-b-b) [inline client/src/models/User.js:208]: The serialize fix is right--toLowerCase(). Why do we want to change the function toProperCase to return lowercase strings?

**Pull request closed without merging**